experimental/packages/otlp-exporter-base/src/configuration/otlp-node-http-env-configuration.ts

```diff
@@ -117,6 +122,55 @@ function getSpecificUrlFromEnv(signalIdentifier: string): string | undefined {
   return appendRootPathToUrlIfNeeded(envUrl);
 }
 
+function readFileFromEnv(
+  signalSpecificEnvVar: string,
+  nonSignalSpecificEnvVar: string,
+  warningMessage: string
+): Buffer | undefined {
+  const signalSpecificPath = process.env[signalSpecificEnvVar]?.trim();
+  const nonSignalSpecificPath = process.env[nonSignalSpecificEnvVar]?.trim();
```

Comment:
`getStringFromEnv()` did not exist yet when I re-wrote that code for the grpc exporters, but we could use it here to align the env var handling (empty string -> undefined). 🙂